src/NSaga/Repository/MySql/MySqlSagaData.cs

```diff
@@ -0,0 +1,16 @@
+﻿using PetaPoco;
```

Comment:
They are a wrapper over the PetaPoco table objects to allow easy configuration of the table objects with different names because the table name is set onto the attributes of SagaData and SagaHeaders objects directly